source/draw/RasterMonochrome.ooc

```diff
@@ -33,6 +33,7 @@ RasterMonochrome: class extends RasterPacked {
 	create: func (size: IntSize2D) -> Image { This new(size) }
 	copy: func -> This {
 		result := This new(this)
+		result _buffer = ByteBuffer new(this _buffer size)
```

Comment:
Hehe, in `RasterImage`,
```ooc
	copy: func ~fromMoreParams (size: FloatSize2D, source: FloatBox2D, upperLeft, upperRight, lowerLeft: FloatPoint2D) -> This {
		result := RasterBgra new(size ceiling() toIntSize2D())
//		TODO: The stuff
		result
	}
```

Maybe I should just leave this as is, open an issue about `copy` being broken and get back to it later.